Makefile

```diff
@@ -88,21 +100,21 @@ install-lua-files:
 	sudo cp viam-cartographer/cartographer/configuration_files/* /usr/local/share/cartographer/lua_files/
 
 test-cpp:
-	cd viam-cartographer && ./scripts/test_cartographer.sh
+	viam-cartographer/build/unit_tests -p -l all
```

Comment:
1. Changed so that we actually show what tests we are running so that when they fail we know where to start.
2. Also I removed the script which wasn't doing anything, other than making the current working directory different than the root directory of the github project, which made it unnecessarily complicated to find artifact files.